packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -35,6 +36,23 @@ describe("calcite-input-date-picker", () => {
     hidden("calcite-input-date-picker");
   });
 
+  describe("reflects", () => {
+    reflects("calcite-input-date-picker ", [
+      {
+        propertyName: "scale",
+        value: "s",
+      },
+      {
+        propertyName: "status",
+        value: "invalid",
+      },
+      {
+        propertyName: "validationIcon",
+        value: true,
+      },
+    ]);
+  });
```

Comment:
@jcfranco not sure why the reflects test for all three of these properties are failing, but I'm going to remove them for now